paper-drawer-panel.html

```diff
@@ -750,16 +779,52 @@
           if (translateX === null) {
             return '';
           }
-
           return this.hasWillChange ? 'translateX(' + translateX + 'px)' :
               'translate3d(' + translateX + 'px, 0, 0)';
         },
 
         _moveDrawer: function(translateX) {
           this.transform(this._transformForTranslateX(translateX), this.$.drawer);
-        }
+        },
 
+        _getDrawerContent: function() {
+          return Polymer.dom(this.$.drawerContent).getDistributedNodes()[0];
+        },
+
+        _getAutoFocusedNode: function() {
+          var drawerContent = this._getDrawerContent(); 
+          return drawerContent.querySelector(this.drawerFocusSelector) || drawerContent;
```

Comment:
Need to use `Polymer.dom(drawerContent).querySelector` so it works the same in Shady DOM and Shadow DOM